htdocs/scss/foundation/_settings.scss

```diff
@@ -636,6 +660,7 @@ $input-font-size: rem-calc(16);
 // $input-disabled-bg: $gainsboro;
 // $input-disabled-cursor: $cursor-default-value;
 // $input-box-shadow: inset 0 1px 2px rgba(0,0,0,0.1);
+$input-include-glowing-effect: true;
```

Comment:
are we deliberately enabling this?

Reply:
I think you already saw this in the per-commit review, but yeah, this is one where we relied on the default and it changed out from under us.